CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Extending the adopted spec, each change should have a link to its corresponding
 
 ## [v1.0.0] - 2019-03-25
 ### Added
+* Fix empty zone list. [#132]
```

Comment:
This entry needs to go under a new release since 1.0.0 was already released. Since this change branch implements a complete patch, we can assume this will be released as v1.0.1.

Reply:
Correction: Unreleased as there is additional content on master which must be logged.